linera-rpc/src/config.rs

```diff
@@ -26,6 +26,10 @@ pub struct CrossChainConfig {
     /// Drop cross-chain messages randomly at the given rate (0 <= rate < 1) (meant for testing).
     #[arg(long = "cross-chain-sender-failure-rate", default_value = "0.0")]
     pub(crate) sender_failure_rate: f32,
+
+    /// How many parallel tasks to spawn for cross-chain message handling RPCs.
+    #[arg(long = "cross-chain-max-tasks", default_value = "10")]
+    pub(crate) max_tasks: usize,
```

Comment:
nit: Perhaps `max_concurrent_tasks` or `max_concurrency` or `concurrency_limit` is more clear?